src/Core/Components/DateTime/FluentDatePicker.razor.cs

```diff
@@ -57,10 +57,10 @@ protected override string? ClassValue
     public EventCallback<MouseEventArgs> OnDoubleClick { get; set; }
 
     /// <summary>
-    /// Gets or sets a value indicating whether today's date is set when double-clicking on the text field of date picker.
+    /// Gets or sets a value which will be set when double-clicking on the text field of date picker.
     /// </summary>
     [Parameter]
-    public bool DoubleClickToToday { get; set; }
+    public DateTime? DoubleClickToDate { get; set; }
```

Comment:
Should we default this to DateTime.Now?

Maybe not for now, but for next version we should consider using `DateTimeOffset` instead of `DateTime`

Reply:
It's default to `null` because it could be an opt-in feature that is "disabled" by default.
Probably it should always be the same date type as the date type in `FluentCalendar`.